### ISSUE TITLE
Create new facility product sitemaps

### ISSUE THREAD

**Issue by andrewlewandowski** [department-of-veterans-affairs/va.gov-team#3634]
We need product sitemaps created for the four new VA facility products:
- VA Medical Center (VAMC) facility system site (all of VHA's VAMC facilities; 144 VAMC systems with ~1,050 health clinics)
- Regional Office (RO) facility system site (all VBA facilities; 60 RO systems with ~300 intake sites)
- Vet Center facility (all of VHA's Vet Centers; 300 Vet Centers)
- VA cemetery (all of NCA's cemeteries; ~150 cemeteries and soldier lots)

The sitemaps should reflect the "lowest common denominator" for each of the facility products (i.e., every required page and where it goes in the product's IA).

The VAMC facility system sitemap should be done first. We hope to have it delivered by the first week of December.

**Comment by mnorthuis** [comment]
@andrewlewandowski converted this to an epic and added the template documentation we discussed as well.  I will create issues for each deliverable as needed.

**Comment by mnorthuis** [comment]
Drafts of VAMC, RO, Vet Center and Cemetery site IA is out there for review and continued refinement.

**Comment by meganhkelley** [comment]
@mnorthuis should this be part of our new super epic #5441 IA of VA.gov — IA Redesign for Online Tools ?

**Comment by mnorthuis** [comment]
No, this is separate.  The IA redesign for online tools is related to benefits and profile.   This work is related to additional content types coming on to the tech stack - and is static content.
